package.json

```diff
@@ -9,14 +9,18 @@
     "jest": "jest",
     "prepare": "npm run build",
     "prepublishOnly": "npm run build",
-    "test": "npm run eslint && npm run build && npm run jest",
+    "test": "npm run eslint && npm run build && npm run jest && npm run testIndividualProjects",
+    "testIndividualProjects": "npm run projectTestPackageJsonConfig",
```

Comment:
It looks like you don't need this separate command anymore since it only calls another

Reply:
I figure there will be more, as we support iopipe.rc down the line...

Reply:
wtm